rules/falco_rules.yaml

```diff
@@ -1881,7 +1884,8 @@
               aws_eks_image_sensitive_mount or
               container.image.repository in (trusted_images) or
               container.image.repository in (falco_sensitive_mount_images) or
-              container.image.repository startswith quay.io/sysdig/)
+              container.image.repository startswith quay.io/sysdig/ or
+              container.image.repository startswith k8scloudprovider/cinder-csi-plugin)
```

Comment:
```suggestion
              container.image.repository=k8scloudprovider/cinder-csi-plugin)
```

Reply:
@darryk10 Thanks this has been fixed now